content/applications/inventory_and_mrp/manufacturing/quality_control/quality_alerts.rst

```diff
@@ -0,0 +1,73 @@
+=====================
+Create quality alerts
+=====================
+
+:doc:`Configuring quality control points <quality_control_points>` is a great way to ensure that
+quality checks are performed at routine stages during specific operations. However, quality issues
+can often appear outside of these scheduled checks. Using Odoo Quality, users can create quality
+alerts for issues that are not detected by automated processes.
+
+Create quality alerts from the quality module
+=============================================
+
+In some situations, it is necessary to manually create quality alerts within the Quality module.
+
+.. example::
+    A helpdesk user who is notified of a product defect by a customer ticket can create an alert that
+    brings the issue to the attention of the relevant quality team.
```

Comment:
This text is indented by one space too many, all text within an admotion block should align with the first letter of the admonition type: `e`xample